### PR TITLE
fixes

### PR DESCRIPTION
**Pull request by tvdboom**
## Related Issue or bug

Adrresses: #2495, #2496, #2497, #2498

Closes #[issue number that will be closed through this PR]

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Code style update (formatting, local variables)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. 

#### Describe if there is any unusual behaviour of your code(Write `NA` if there isn't)

A clear and concise description of it.

## Checklist:

- [ ] My code follows the style guidelines of this project.
- [ ] I have performed a self-review of my own code.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] I have made corresponding changes to the documentation.
- [ ] My changes generate no new warnings.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] New and existing unit tests pass locally with my changes.
- [ ] Any dependent changes have been merged and published in downstream modules.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -38,3 +38,4 @@ demo4.py
 dask*/
 /.venv
 /.devcontainer
+/*.ipynb
```
[the remark this answers is not in the document]
Comment (tvdboom) [reply .gitignore:41]: well, that's exactly why I put it there. Why do we have examples in the main directory. Shouldn't those be stored somewhere else? Like a `example` dir for example

[the remark this answers is not in the document]
Comment (ngupta23) [reply .gitignore:41]: Yes, I have to move the time series notebooks to the `examples` folder. It is a TODO before the RC release.
But do we need ipynb in gitignore?

File: pycaret/classification/oop.py [hunk 2]
```diff
@@ -339,21 +339,22 @@ def setup(
         container = []
         container.append(["Session id", self.seed])
         container.append(["Target", self.target_param])
-        container.append(["Target type", "classification"])
+        container.append(["Target type", "Binary" if self.y.nunique() == 2 else "Multiclass"])
```
Comment (Yard1) [inline pycaret/classification/oop.py:342]: I wonder if we can encapsulate the binary check in a separate function

File: pycaret/classification/oop.py [hunk 3]
```diff
@@ -339,21 +339,22 @@ def setup(
...
+        container.append(["Original data shape", self.dataset.shape])
+        container.append(["Transformed data shape", self.dataset_transformed.shape])
+        container.append(["Transformed train set shape", self.train_transformed.shape])
+        container.append(["Transformed test set shape", self.test_transformed.shape])
         for fx, cols in self._fxs.items():
             if len(cols) > 0:
                 container.append([f"{fx} features", len(cols)])
         if self.data.isna().sum().sum():
-            container.append(["Missing Values", self.data.isna().sum().sum()])
+            container.append(["Missing Values", f"{round(self.data.isna().sum().sum() / self.data.size, 2)}%"])
```
Comment (Yard1) [inline pycaret/classification/oop.py:357]: Maybe "Rows with missing values"?

**Review by Yard1 — APPROVED** [review]
Looks good, two nits

File: pycaret/classification/oop.py (hunk 2 above)
Comment (tvdboom) [reply pycaret/classification/oop.py:342]: in utils.py? easy enough...

File: .gitignore (hunk 1 above)
[the remark this answers is not in the document]
Comment (ngupta23) [reply .gitignore]: ok, sounds good. Are you going to remove `nb_test.ipynb` from the branch?

**Pull request merged**